packages/core/src/domain/patient-demographics.ts

```diff
@@ -35,3 +35,16 @@ export type LinkDemographicsComparison = Partial<LinkDemographics>;
 export type LinkDemographicsHistory = {
   [key: string]: LinkDemographics[];
 };
+
+export type GenderAtBirth = "M" | "F" | "UK";
+
+export function mapGenderCode(gender: string | undefined): GenderAtBirth {
+  switch (gender?.toLowerCase().trim()) {
+    case "male" || "m":
+      return "M";
+    case "female" || "f":
+      return "F";
+    default:
+      return "UK";
+  }
```

Comment:
second thought. This is supposed to be converting fhir to CDA, but f and m are not valid values in fhir.

Reply:
and also the name of the function should convey that its converting from fhir to CDA

Reply:
lol true.. But I'll keep a copy of this function here for future use :P